source/server/getting-started/jre.rst

```diff
@@ -19,11 +19,16 @@ Java, and it should work just as well, if not better. However, it's also possibl
 Linux <https://www.oracle.com/technetwork/java/javase/downloads/index.html>`__, but be aware that many Java-dependent
 Linux packages will still install OpenJDK anyway.
 
+You may have to configure the path to the JRE/JDK in your 
+
+* launcher using ``Profile`` (``Advanced Options``) -> ``<Your Profile>`` -> ``Java-Executable`` (``.../java.exe``)
+* server using either the ``PATH`` variable or using the full path to your java executable
+
 32-bit vs. 64-bit
 ~~~~~~~~~~~~~~~~~
 
 If your computer supports it, you should use 64-bit versions of Java whenever possible. The Java installers from the
-linked website should detect whether your computer is ready for 64-bit.
+`this website <https://java.com/en/download/>`__ should detect whether your computer is ready for 64-bit.
```

Comment:
probably wiser to use https://java.com/en/download/manual.jsp

Reply:
Not sure about that, there is the selection, but it does not really help you with selecting 32 or 64 bit.

Reply:
Some browsers can handle the auto-detection, and some (eg. *Pale Moon*) cannot. The manual downloads page is a good place to make sure you *get* a 64-bit version, but you can also have multiple versions of Java installed so it's no guarantee that it'll be the one used by software. The Java control panel will give you the info in Windows, but I'm not sure about Mac or Linux systems.